File: api/code-samples/python/debug/profiling/mutexProfile.py
import json
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

fileName = "mutex.profile"
seconds = 10

sdk = OpenSDK(host)
debug_response = sdk.debug.mutex_profile(fileName, seconds)

print(json.loads(debug_response.response.data))

File: web3rpc/rpc-specs/code-samples/javascript/debug/profilling/mutexProfile.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
  const file = "mutex.profile";
  const seconds = 10;

  provider.debug
    .mutexProfile(file, seconds, {}, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();
